.github/workflows/docker.yml

```diff
@@ -17,7 +18,7 @@ jobs:
           - "6.7.0"
           - "7.2.0"
```

Comment:
If you're already updating versions here, would you also update Ansible itself? I think we should drop 6.7.0, replace 7.2.0 with 7.7.0 and add 8.3.0